### PR TITLE
add -i to specify the location of the values.yaml

### PR DESCRIPTION
**Pull request by itdove**

### CODE REVIEW

File: create-cluster/deploy.sh [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+  echo "-i: the path to the values.yaml, default values.yaml"
+  echo "-o: output-file: generate an output-file instead of applying"
   echo "-d: When set the cluster will be destroyed"
   echo "-v: verbose level"
   echo "-h: this help"
   exit 0
 fi
-
-PARAMS="$(applier -d params.yaml -values values.yaml -o /dev/stdout -s)"
+if [ -z ${INPUT+x} ]
```
Comment (chrisahl) [inline create-cluster/deploy.sh]: INPUT or IN?

File: create-cluster/deploy.sh [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
+then
+  INPUT="-values values.yaml"
```
Comment (chrisahl) [inline create-cluster/deploy.sh]: INPUT or IN?

**Review by chrisahl — CHANGES_REQUESTED** [review]
Please look at my comments.

File: create-cluster/deploy.sh (hunk 1 above)
Comment (itdove) [reply create-cluster/deploy.sh]: Sorry IN

**Review by chrisahl — DISMISSED** [review]

**Review by chrisahl — APPROVED** [review]

**Pull request merged**